static/js/containers/DashboardPage.js

```diff
@@ -294,6 +297,17 @@ class DashboardPage extends React.Component {
     this.fetchCoupons()
     this.handleOrderStatus()
     this.fetchDiscussionsFrontpage()
+    this.fetchUserProfile()
+  }
+
+  fetchUserProfile = () => {
+    const { financialAid: { fetchError }, dispatch } = this.props
+    if (fetchError && fetchError.message === "Profile is not complete") {
+      dispatch(clearProfile(SETTINGS.user.username))
+      this.context.router.push(`/profile/`)
+      dispatch(hideDialog(INCOME_DIALOG))
+      dispatch(clearCalculatorEdit())
```

Comment:
it might be good to put up a toast message here (similar to what we do in `App.js`) that says something like 'hey, fill out your profile'

Reply:
oh nvm the toast message in `App.js` seems to firing anyway